### PR TITLE
Add SubtleAlert RPC documentation

### PR DESCRIPTION
**Pull request by jacobkeeler**
Adds documentation and diagrams for `UI.SubtleAlert` and `UI.OnSubtleAlertPressed`.

### CODE REVIEW

File: docs/UI/SubtleAlert/index.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+### Sequence Diagrams
+
+|||
+SubtleAlert closed by DEFAULT_ACTION
+![SubtleAlert](./assets/AlertDefaultAction.png)
+|||
+
+|||
+SubtleAlert closed by STEAL_FOCUS
+![SubtleAlert](./assets/AlertStealFocus.png)
```
Comment (ShobhitAd) [inline docs/UI/SubtleAlert/index.md:69]:
```suggestion
![SubtleAlert](./assets/SubtleAlertStealFocus.png)
```

File: docs/UI/SubtleAlert/index.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+|||
+
+|||
+SubtleAlert Aborted by <abbr title="Voice Recognition">VR</abbr> Session
+![SubtleAlert](./assets/AlertAborted.png)
```
Comment (ShobhitAd) [inline docs/UI/SubtleAlert/index.md:74]:
```suggestion
![SubtleAlert](./assets/SubtleAlertAborted.png)
```

File: docs/UI/SubtleAlert/index.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+|||
+
+|||
+SubtleAlert Rejected
+![SubtleAlert](./assets/AlertRejected.png)
```
Comment (ShobhitAd) [inline docs/UI/SubtleAlert/index.md:79]:
```suggestion
![SubtleAlert](./assets/SubtleAlertRejected.png)
```

File: docs/UI/SubtleAlert/index.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+|||
+
+|||
+SubtleAlert BOTH UI Closed before <abbr title="Text To Speech">TTS</abbr> finishes Speaking
+![SubtleAlert](./assets/AlertTTS.png)
```
Comment (ShobhitAd) [inline docs/UI/SubtleAlert/index.md:84]:
```suggestion
![SubtleAlert](./assets/SubtleAlertTTSAborted.png)
```

File: config.json [hunk 5]
```diff
@@ -405,6 +408,9 @@
             {
               "name": "Slider"
             },
+            {
+              "name": "SubtleAlert"
+            },
```
[the remark this answers is not in the document]
Comment (jacobkeeler) [reply config.json:413]: Sadly things aren't too consistent on this end, most RPCs are alphabetical, but there are some odd exceptions, I'm fine with moving this though

**Pull request merged**